diff --git a/src/atv.cpp b/src/atv.cpp
index bda6ab3..b544223 100644
--- a/src/atv.cpp
+++ b/src/atv.cpp
@@ -158,7 +158,7 @@ void atv::SetDmaAlgo()
     }
     cbp--;
     cbp->next = mem_virt_to_phys(cbarray); // We loop to the first CB
-    dbg_printf(1, "Last cbp :  %d \n", ((unsigned int)(cbp) - (unsigned int)(cbarray)) / sizeof(dma_cb_t));
+    dbg_printf(1, "Last cbp :  %d \n", ((unsigned long)(cbp) - (unsigned long)(cbarray)) / sizeof(dma_cb_t));
 }
 
 void atv::SetFrame(unsigned char *Luminance, size_t Lines)
